test/e2e/renderers/checkboxRenderer.spec.js

```diff
@@ -818,6 +818,84 @@ describe('CheckboxRenderer', () => {
     expect(getCell(0, 0).querySelector('label').lastChild.textContent).toEqual('myLabel');
   });
 
+  it('should remove checkbox and do not add #bad-value# content after cut action', () => {
+    const hot = handsontable({
+      data: [
+        { car: 'Mercedes A 160', available: true, comesInBlack: 'yes' },
+        { car: 'Citroen C4 Coupe', available: false, comesInBlack: '' },
+        { car: 'Audi A4 Avant', available: true, comesInBlack: 'no' },
+      ],
+      colHeaders: ['Car model', 'Accepted', 'Comes in black'],
+      columns: [
+        {
+          data: 'car'
+        },
+        {
+          data: 'available',
+          type: 'checkbox',
+          label: {
+            position: 'after',
+            property: 'car'
+          },
+        },
+        {
+          data: 'comesInBlack',
+          type: 'checkbox',
+          checkedTemplate: 'yes',
+          uncheckedTemplate: 'no',
+        },
+      ],
+    });
+    const cutEvent = getClipboardEvent('cut');
+    const plugin = hot.getPlugin('CopyPaste');
+    const td = hot.getCell(0, 2);
+
+    selectCell(0, 2);
```

Comment:
Could you select all cells and verify how it works with both cases of using the checkbox type?

Reply:
Yes, I wil try.